prowler/providers/aws/services/cloudwatch/cloudwatch_log_group_retention_policy_specific_days_enabled/cloudwatch_log_group_retention_policy_specific_days_enabled.metadata.json

```
@@ -15,9 +15,9 @@
   "RelatedUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Logs.html",
   "Remediation": {
     "Code": {
-      "CLI": "https://docs.prowler.com/checks/aws/logging-policies/logging_13#cli-command",
+      "CLI": "put-retention-policy --log-group-name <value> --retention-in-days <value> [--cli-input-json <value>] [--generate-cli-skeleton <value>]",
```

we are missing here `aws logs` :
```suggestion
      "CLI": "aws logs put-retention-policy --log-group-name <value> --retention-in-days <value>",
```